include/picongpu/particles/Particles.hpp

```diff
@@ -107,7 +110,7 @@ namespace picongpu
                 // fallback if the species has not defined the alias boundaryCondition
                 pmacc::HandleGuardRegion<
                     pmacc::particles::policies::ExchangeParticles,
-                    particles::boundary::CallPluginsAndDeleteParticles>>::type>;
+                    pmacc::particles::policies::DeleteParticles>>::type>;
```

Comment:
same here, should be an empty functor.

Reply:
Yes, I didn't have time to implement it yesterday